Documentation/usingupm.md

```diff
@@ -0,0 +1,74 @@
+# Mixed Reality Toolkit and Unity Package Manager
+
+Starting with version 2.5.0, the Microsoft Mixed Reality Toolkit is available using the Unity Package Manager (UPM), on Unity 2019.4 and newer. Using UPM, the MRTK gains dependency management and a simple update path, as will be described in the following sections.
+
+## Installing Mixed Reality features using the Unity Package Manager
+
+The Unity Package Mananger uses a manifest file (manifest.json) to determine which packages to install and the registries (servers) from which they can be installed.
+
+> [!Note]
+> As of version 2.5.0 of the MRTK, initial registration of the server and packages is a per-project, manual procedure, please read the following sections for detailed instructions.
+
+### Registering the Mixed Reality component server
+
+For each project that will be using the Microsoft Mixed Realuty Toolkit, the `manifest.json` file (in the Packages folder) will need to have the Mixed Reality scoped registry added. The following illustrate how to properly modify `manfest.json` to support Mixed Reality.
```

Comment:
Realuty?!?!?

Can you run these changes using this:

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

I think there were a bunch of other misspellings in the other file, would be great if you could run this to avoid having reviewers do a manual spell checking pass.

Reply:
haha! typing too fast 😺

Reply:
fixed